### ISSUE TITLE
Weird peerDependency issue. 

### ISSUE THREAD

**Issue by JustFly1984** [graphql/graphql-relay-js#220]
I'm trying to update my app to graphql@14.0.2, but getting this error from `npm i`:

```
npm WARN graphql-relay@0.5.5 requires a peer of graphql@^0.5.0 || ^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
```

I had forked the repo, and I see correct peerDependency statement in package.json,including graphql@^14.0.2 and version of   graphql-relay@0.5.5

Could be there an issue with publishing?

**Comment by JustFly1984** [comment]
Is there any movement about new release?

**Comment by kassens** [comment]
Should be fixed in 0.6.0 now!

If someone want's to take a more active maintenance of this repo, please reach out to me on Twitter @kassens.
